competition-simulator.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: page
+title: Competition Simulator
+---
+
+Competition Simulator
+============
+
+![]({{ site.baseurl }}/resources/competition-simulator/arena.png)
+
+For SR2020, a virtual competition will be run using a brand new simulator.
+
+<div class="info">
+  This is different to the <a href="/docs/programming/simulator/">regular simulator</a>.
+</div>
+
+## Installation
+
+### Prerequisites
+
+This simulation is based in [Webots](https://cyberbotics.com/#download), which will need to be downloaded and installed.
+
+You will also need Python installed. The simulator supports >=3.5. Additional external libraries are not supported.
+
+### Installing the simulation
+
+1. [Download the simulation](https://github.com/srobo/competition-simulator/archive/0.1.0.zip), and unzip it somewhere on your computer.
+2. Using the Webots IDE, open the `worlds/Arena.wbt` file.
+
+You may receive a warning about your computer's GPU not being good enough, which can be ignored.
+
+#### Changing your version of Python
+
+If webots is picking up the incorrect version of Python, you'll need to change it. This can be done using `Tools > Preferences > General > Python command`. You'll need to ensure a matching version of Python is installed.
```

Comment:
I think we worked out that we can avoid this by ensuring there's a suitable shebang in the controller file?

Reply:
I'm not familiar with how that works on windows. I think being explicit like this is still the better option

Reply:
Shebangs don't normally work on Windows, though I don't recall whether Webots implements that limitation. It's possible that it would respect the shebang anyway. I do suggest we check it before removing this though.